engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,7 +162,7 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRotation;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                if (!playerCamera.isTracked()) {
+                if (!config.getRendering().isVrSupport()) {
```

Comment:
It's possible, as you can see in this commit (which I've left out of this branch because I don't know if it's the best thing to do):

https://github.com/indianajohn/Terasology/commit/6bca73285992fa5a91849a9ce89245a4282c2a9b

Anyway, if we think we should add that I can push it to this branch.